src/java.xml/share/classes/com/sun/org/apache/xml/internal/serializer/dom3/DOMConstants.java

```diff
@@ -113,6 +118,18 @@
     // The xerces serializer specific 'omit-xml-declaration' property used in LSSerializer
     public static final String S_XML_VERSION = "xml-version";
 
+    /**
+     * Indicates that the serializer should treat the output as a standalone document.
+     * The JDK specific standalone property controls whether a newline should be
+     * added after the XML header.
+     *
+     * @see similar property ORACLE_IS_STANDALONE in OutputPropertiesFactory.
+     */
+    public static final String S_IS_STANDALONE = "jdk-is-standalone";
+
+    // Corresponding System property
+    public static final String SP_IS_STANDALONE = "jdk.xml.isStandalone";
```

Comment:
Should this system property key be described somewhere in the javadoc, as an Oracle implSpec? Otherwise, how the user would know this switch?

Reply:
So far as with the current practice, we would document JDK specific properties in the Release Notes (refer to the Release Note sub-task for this particular property). We also created tutorials for the ones that are within a major topic (e.g. security). Plus, even if we'd want to add an implSpec, we wouldn't be able to since the DOM's license is quite restrictive.

Reply:
OK. Thanks for the explanation.

Reply:
Note: If such properties needed to be documented - they could also be documented as JDK implementation specific properties in the module-info file. This is what we have started doing for some of the properties supported by JNDI providers, for instance.